style.css

```diff
@@ -0,0 +1,505 @@
+  margin-left: 20px;
+}
+
+.btn-resume {
+  font-size: 15px;
+  font-weight: 400;
+}
+
+.accordion {
+  margin-top: 60px;
+}
+
+.accordion-container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 375px;
+}
+
+.accordion li {
+  list-style: none;
+}
+
+.skill--description {
+  font-size: 20px;
+  font-family: var(--font-family);
+  font-weight: 500;
+  line-height: 24px;
+  text-decoration: none;
+  color: black;
+}
+
+.skill--description-link {
+  text-decoration: none;
+  margin-left: 36px;
+}
+
+.skill {
+  display: flex;
+  align-items: center;
+  margin: 36px 36px 12px 36px;
+  width: 303px;
+  height: 66.72px;
+  background-color: #f7f7f9;
+}
+
+.skill-image {
+  margin: 9px 12px 9.72px;
+  height: 48px;
+  width: 48px;
+  object-fit: cover;
+}
+
+.skill-name {
+  font-family: var(--font-family);
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #253858;
+}
+
+.chevron-icons {
+  cursor: pointer;
+  margin-right: 64px;
+}
+
+.frameworks-container,
+.skills-container {
+  width: 375px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+hr {
+  width: 327px;
+  height: 1px;
+  background-color: #dfe1e6;
+  margin: 0 auto;
+}
+
+.framework--title-link,
+.skills--title-link {
+  text-decoration: none;
+  font-size: 20px;
+  font-family: var(--font-family);
+  font-weight: 500;
+  line-height: 24px;
+  color: black;
+  margin: 36px 36px;
+}
+
+.contact-me-main-container {
+  background-color: #fff;
+  padding-top: 50px;
+}
+
+.contact-me-section {
+  height: 779px;
+  background-image: url(images/background/contact-me-background.svg);
+}
+
+.contact-me-header {
+  padding-top: 105px;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  font-family: var(--font-family);
+}
+
+.contact-me-p {
+  align-items: center;
+  text-align: center;
+  color: #ebebff;
+  font-weight: 400;
+  font-size: 20px;
+  font-family: var(--font-family);
+  margin-top: 20px;
+}
+
+.form-div {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 45px;
+}
+
+.form-input-name,
+.form-input-email {
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  width: 327px;
+  height: 48px;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  font-family: var(--font-family);
+  color: #172b4d;
+  align-items: center;
+  cursor: pointer;
+  padding: 10px;
+}
+
+.form-input-email {
+  margin-top: 10px;
+}
+
+.text-message {
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #b3bac5;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  width: 327px;
+  height: 170px;
+  margin-top: 10px;
+  padding: 10px;
+  font-family: var(--font-family);
+}
+
+.submit-btn-div {
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+}
+
+.Get-In-Touch-btn {
+  width: 140px;
+  height: 48px;
+  font-weight: 500;
+  font-size: 17px;
```

Comment:
- [ ] Kindly add button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template:
![image](https://user-images.githubusercontent.com/91486791/185995804-9309194b-c70c-4848-b0c8-e06540ed417c.png)

Reply:
Hi, I think the style is not supposed to be applied to the ***Get In Touch*** button. please recheck.

Reply:
![image](https://user-images.githubusercontent.com/91486791/186007676-cb281793-a6a9-41b5-b3e7-67dcd91b59f0.png)

Reply:
Hi @samuelmoiwa I think you missed that point in the project requirements. Here's a screenshot: ![image](https://user-images.githubusercontent.com/91486791/186007676-cb281793-a6a9-41b5-b3e7-67dcd91b59f0.png)

Reply:
but if the hover style is added to the button, it blends in with the background colour making the button invisible, don't know, but I think this button style is only applied to the languages and see project button.